### ISSUE TITLE
frcnn_cython can't installed on Windows

### ISSUE THREAD

**Issue by gwsbhqt** [deepinsight/insightface#294]
I've tried to change the real path of nvvc and lib64, and I've removed default_compiler_so, but still can't be installed. Is SSH really unavailable on Windows? Did anyone else meet or solve the problem? Thank you!

**Comment by nttstar** [comment]
Windows is not recommended.

**Comment by gwsbhqt** [comment]
@nttstar Thank you very much anyway!

**Comment by gwsbhqt** [comment]
@nttstar I seem to remember that you said that the GA model was trained by Asians, but when I actually tested myself and the people around me, the misrecognition rate was still quite high. The age misidentification was acceptable, but the gender misrecognition could not be understanding. Is there such a situation when you test?

**Comment by nttstar** [comment]
Did you use the same alignment method as arcface did?

**Comment by gwsbhqt** [comment]
@nttstar I cut the image using Mtcnn with rotation correction to 112px, and then handed them to InsightFace GA recognition.

**Comment by gwsbhqt** [comment]
@nttstar Although I know that InsightFace has Mtcnn alignment, it seems that InsightFace's own Mtcnn does not do rotation correction, some of the skew face recognition rate will be lower, and the rate of recognition is much higher after the rotation correction. So before I handed it to InsightFace, I used a custom Mtcnn to correct the rotation angle and cut it into 112px. I don't know if this behavior is correct.
